frontend/src/components/Router.tsx

```diff
@@ -159,23 +159,23 @@ const Router: React.FC<RouterProps> = ({ configs }) => {
   const routes: RouteConfig[] = configs || [
     { path: RoutePage.START, Component: GettingStarted },
     {
-      Component: ArchivedExperimentsAndRuns,
+      Component: AllRunsAndArchive,
       path: RoutePage.ARCHIVED_RUNS,
```

Comment:
I think in the new structure, it makes more sense to change the route from `/archive/runs` to `/runs/archive`, what do you think? Maybe we can do it in a separate PR, because route change means we might need to set up redirect from the old route to the new route to ease the migration if people have built systems that take KFP UI URL.

Reply:
I definitely agree on this and actually wanted to propose it as a future improvement. I'll think about it and propose next steps